astronify/series.py

```diff
@@ -102,15 +116,15 @@ def sonify(self):
         # NOTE: This assumes the times in the time_col are an astropy Time object
```

Comment:
This comment can be removed now that the Time vs. float is handled in the object creation.